ipld/resolver/src/behaviour/membership.rs

```diff
@@ -239,3 +316,22 @@ impl NetworkBehaviour for Behaviour {
         Poll::Pending
     }
 }
+
+// Forest has Filecoin specific values copied from Lotus. Not sure what values to use,
+// so I'll leave everything on default for now. Or maybe they should be left empty?
```

Comment:
I think default is the conservative decision here, once we start testing we should immediately see if this is a good choice for as first deployment.